### DISCUSSION TITLE
Python Named Access Full List

### DISCUSSION THREAD

**Discussion by rohit-kumar-j** [google-deepmind/mujoco#416] [General]
To access properties from the model/data is this the full page for the accessors:[python/mujoco/indexer_xmacro.h](https://github.com/deepmind/mujoco/blob/64bc6d27b242d5c5683b4a03dd36f1844b910742/python/mujoco/indexer_xmacro.h)?
for `model.xml_attr("property_name").property` and `data.xml_attr("property_name").property`

Eg:
```
model.actuator("actuator_name").forcerange  #  trntype, dyntype, gaintype, biastype, etc...
data.actuator("actuator_name").ctrl  # ctrl, length, moment, velocity, force
```

**Comment by saran-t** [comment]
Yes.

**Reply by rohit-kumar-j** [reply]
Thanks!
